src/contracts/middleware/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -275,6 +275,11 @@ contract BLSRegistryCoordinatorWithIndices is Initializable, IBLSRegistryCoordin
         _deregisterOperatorWithCoordinator(msg.sender, quorumNumbers, pubkey, operatorIdsToSwap);
     }
 
+    function updateSocket(string memory socket) external {
+        require(_operators[msg.sender].status == OperatorStatus.REGISTERED, "BLSIndexRegistryCoordinator.updateSocket: operator is not registered");
```

Comment:
oh also need the correct contract name here

Reply:
went ahead and updated all the revert strings